tests/test_compute.py

```diff
@@ -224,9 +229,27 @@ def compute_shader(
     with raises(ValueError):  # output_arrays shape invalid
         compute_with_buffers({0: in1}, {0: ("i",)}, compute_shader)
     with raises(ValueError):  # output_arrays shape invalid
-        compute_with_buffers({0: in1}, {0: (0, "i",)}, compute_shader)
+        compute_with_buffers(
+            {0: in1},
+            {
+                0: (
+                    0,
+                    "i",
+                )
+            },
+            compute_shader,
+        )
     with raises(ValueError):  # output_arrays shape invalid
-        compute_with_buffers({0: in1}, {0: (-1, "i",)}, compute_shader)
+        compute_with_buffers(
+            {0: in1},
+            {
+                0: (
+                    -1,
+                    "i",
+                )
+            },
+            compute_shader,
+        )
```

Comment:
How is this supposed to be more readable? :(

Reply:
Black does not make things more readable... ;) it just enforces rules consistently

Reply:
I just wish it would enforce rules that make it more readable ;)

Reply:
This is just silly

Reply:
They should include a sillyness factor in their algorithm :)